sidekick/lib/src/init/init_command.dart

```diff
@@ -222,13 +224,34 @@ class InitCommand extends Command {
       await makeExecutable(entrypoint);
     }
 
-    // For now, we install the flutter wrapper to get a dart runtime.
-    // TODO Add dart runtime so that dart packages can use sidekick without flutter
-    final flutterw = await installFlutterWrapper(entrypointDir);
+    // Install flutterw when a Flutter project is detected
+    final flutterPackages = [if (mainProject != null) mainProject, ...packages]
+        .filter((package) => package.isFlutterPackage);
 
+    if (flutterPackages.isNotEmpty) {
+      print('We detected Flutter packages in your project:');
+      for (final package in flutterPackages) {
+        print('  - ${package.name} '
+            'at ${relative(package.root.path, from: repoRoot.absolute.path)}');
+      }
+
+      print('\n\n'
+          '${dcli.green('Do you want pin the Flutter version of this project with flutterw?\n')}'
+          'https://github.com/passsy/flutter_wrapper\n\n'
+          'This allows you to use the `$cliName dart` and `$cliName flutter` commands\n');
+      final confirmFlutterwInstall = dcli.confirm(
```

Comment:
That's true for the first install. But when upgrading (which is currently also done with `sidekick init`) it would be annoying when it removed the `dart` and `flutter` commands, even if you already installed `flutterw`.

I'd wait for the plugin system, then we can add those commands with the `flutterw` and `fvm` plugins